Solutions/Business Email Compromise - Financial Fraud/Analytic Rules/SuspiciousAccessOfBECRelatedDocuments.yaml

```diff
@@ -46,10 +46,14 @@ query: |
 entityMappings:
   - entityType: Account
     fieldMappings:
+      - identifier: FullName
+        columnName: User
       - identifier: Name
         columnName: Name
       - identifier: UPNSuffix
```

Comment:
I am not sure this will come through as a UPN from the imFileEvent function.  You should confirm if this is in the format of NTDomain\UserName or UPN. If it comes in both, then you can handle both in the mappings also.